optimum/exporters/onnx/utils.py

```diff
@@ -190,6 +232,18 @@ def _get_submodels_and_onnx_configs(
     preprocessors: list[Any] | None = None,
     model_kwargs: dict | None = None,
 ):
+    if model.config.model_type == "metaclip_2":
+        export_config_constructor = TasksManager.get_exporter_config_constructor(
+            model=model, exporter="onnx", task=task, library_name="transformers"
+        )
+        export_config = export_config_constructor(
+            model.config,
+            int_dtype=int_dtype,
+            float_dtype=float_dtype,
+            preprocessors=preprocessors,
+        )
+        export_config.variant = _variant
```

Comment:
please add the same logging message in https://github.com/huggingface/optimum/blob/a813c95ac088c401547fe15e7a68ac5c6f00f9a7/optimum/exporters/utils.py#L579C1-L582C118